@xen-orchestra/lite/src/components/vm/VmsActionsBar.vue

```diff
@@ -10,21 +10,43 @@
       <UiButton :active="isOpen" :icon="faEllipsis" transparent @click="open" />
     </template>
 
-    <MenuItem :icon="faPowerOff">{{ $t("change-power-state") }}</MenuItem>
-    <MenuItem :icon="faRoute">{{ $t("migrate") }}</MenuItem>
-    <MenuItem :icon="faCopy">{{ $t("copy") }}</MenuItem>
-    <MenuItem :icon="faEdit">{{ $t("edit-config") }}</MenuItem>
-    <MenuItem :icon="faCamera">{{ $t("snapshot") }}</MenuItem>
-    <MenuItem :icon="faBox">{{ $t("backup") }}</MenuItem>
-    <MenuItem :icon="faTrashCan">{{ $t("delete") }}</MenuItem>
+    <MenuItem :icon="faPowerOff" v-tooltip="$t('coming-soon')">
+      {{ $t("change-power-state") }}
+    </MenuItem>
+    <MenuItem :icon="faRoute" v-tooltip="$t('coming-soon')">{{
+      $t("migrate")
+    }}</MenuItem>
+    <MenuItem :icon="faCopy" v-tooltip="$t('coming-soon')">{{
+      $t("copy")
+    }}</MenuItem>
+    <MenuItem :icon="faEdit" v-tooltip="$t('coming-soon')">{{
+      $t("edit-config")
+    }}</MenuItem>
+    <MenuItem :icon="faCamera" v-tooltip="$t('coming-soon')">{{
+      $t("snapshot")
+    }}</MenuItem>
+    <MenuItem :icon="faTrashCan" v-tooltip="$t('coming-soon')">{{
+      $t("delete")
+    }}</MenuItem>
     <MenuItem :icon="faFileExport">
```

Comment:
Shouldn't we also add a tooltip here?

Reply:
I was thinking about it at first but I think it's better to put tooltips on *features* that aren't enabled yet. But this is not a feature, it's a menu with 3 (almost) independent features inside.